doc/build-openbsd.md

```diff
@@ -63,50 +62,50 @@ export BDB_PREFIX="/path/to/bitcoin/depends/x86_64-unknown-openbsd"
 Bitcoin Core includes a GUI built with the cross-platform Qt Framework. To compile the GUI, Qt 5 is required.
 
 ```bash
-pkg_add qt5
+pkg_add qtbase qttools
 ```
 
-## Building Bitcoin Core
-
-**Important**: Use `gmake` (the non-GNU `make` will exit with an error).
+#### Test Suite Dependencies
+There is an included test suite that is useful for testing code changes when developing.
+To run the test suite (recommended), you will need to have Python 3 installed:
 
-Preparation:
 ```bash
+pkg_add install python  # Select the newest version of the package.
+```
 
-# Adapt the following for the version you installed (major.minor only):
-export AUTOCONF_VERSION=2.71
-export AUTOMAKE_VERSION=1.16
+## Building Bitcoin Core
 
-./autogen.sh
+Prepare a build directory:
+```bash
+mkdir build
+cd build
 ```
 
 ### 1. Configuration
 
 There are many ways to configure Bitcoin Core, here are a few common examples:
 
 ##### Descriptor Wallet and GUI:
-This enables the GUI and descriptor wallet support, assuming `sqlite` and `qt5` are installed.
+This enables the GUI and descriptor wallet support, assuming SQLite and Qt 5 are installed.
 
 ```bash
-./configure MAKE=gmake
+cmake -S .. -DWITH_SQLITE=ON -DWITH_GUI=Qt5
```

Comment:
Our suggestions boil down to:

```
mkdir build
cd build
cmake -S ..
cmake --build .
```

It just occurred to me that `cmake` will create the build directory if it does not exist, so we don't need to `mkdir` it. We don't need to `cd` into it either. Consider the following instead:

```
cmake -B build
cmake --build build
```

Reply:
The point is the `mkdir build && cd build` invocation guaranties the _clean_ build environment. Another benefit of `cd build` is that there is no need to specify the `--test-dir` option when running `ctest`.

Reply:
It's my observation that most documentation seems to prefer `mkdir` followed by `cd` over using `-B`. I often use the shorter route but I think for documentation perhaps it should be be the longer slightly safer route?

Reply:
I think that's mostly because a lot of docs were written before CMake had the ability to automatically create the dirs, and then it's just never been updated.

Reply:
[Reworked](https://github.com/hebasto/bitcoin/pull/167#issuecomment-2077532788).